### ISSUE TITLE
Allow sorting of items

### ISSUE THREAD

**Issue by avently** [orgzly/orgzly-android#13]
Orgzly is the best app in its category. And one of task where Orgzly is useful is learning new languages. So sometimes I want to sort items under certain label because it is easy to find words I should learn. For example, I have a category "New words":
- New words
-- day
-- work
-- example
-- temperature

In this example would cool to have a possibility to sort all items under "New words" alphabetically (or any other type of sorting you can imagine).

**Comment by sokolgeo** [comment]
I am second to this. Can we please add sorting notes by title? Comes really handy for shopping lists.

So the following to be supported:

`o.title'

**Comment by nevenz** [comment]
I think this issue is about reordering sub-notes of a specified note in a notebook?

`o.title` would be sort order in search results (which is a good suggestion, but a separate issue which can be opened).

**Comment by avently** [comment]
@nevenz 
> I think this issue is about reordering sub-notes of a specified note in a notebook?

Yes. Does anything changed since the issue creation? Maybe now the sorting feature can be implemented?

**Comment by nevenz** [comment]
> Does anything changed since the issue creation? Maybe now the sorting feature can be implemented?

I'm not aware of anyone starting to work on it yet.

**Comment by dorneanu** [comment]
Hi!

I'm also looking forward to this one. I need this feature since I want to sort (sub-)trees on the scheduled time. This might be a workaround to the calendar view since you can always insert new items/appointments and the sort the whole tree to see if there are any conflicts let's say between future appointments.
